src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/InfoFieldAnnotation.java

```diff
@@ -17,7 +17,7 @@
 
     /**
      * Computes the annotation for the given variant and the likelihoods per read.
-     * Returns a map from annotation keys to values.
+     * Returns a map from annotation keys to values or null if no annotation is to be added.
```

Comment:
Not sure you've addressed this @akiezun -- is this doc accurate, or should it say empty map instead of null?